dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:20
```

Comment:
Can you look at this Dockerfile:
https://github.com/fiaisis/plotting-service/blob/main/data-viewer/Dockerfile

Specific points around disabling the optional next telemetry, the node_env, creating user and user group, so that it isnt run as root.

You should also look at the `CMD` we do not want to run the container with a dev server.

The multistep build also can help us if we ever are urgently debugging and need to keep manually rebuilding and pushing the image for staging.